source/sdk/threading/Thread.ooc

```diff
@@ -165,6 +172,23 @@ Mutex: abstract class {
 
 }
 
+MutexUnsafe: class extends Mutex {
```

Comment:
Hence the name :D
It is used to implement reference counters which do not use a real mutex lock but still needs the mutex interface for the sake of simplicity.